lib/UtapiReindex.js

```diff
@@ -61,6 +61,8 @@ class UtapiReindex {
             this._log = new werelogs.Logger('UtapiReindex', { level, dump });
         }
 
+        this._onlyCountLatestWhenObjectLocked = (config && config.onlyCountLatestWhenObjectLocked === true);
```

Comment:
Question from a Node noob: would `(config?.onlyCountLatestWhenObjectLocked === true)` achieve the same thing?

Reply:
Yeah it would give the same result. The reason I using the older `(config && config...)` is that the linter version used doesn't support the `?` syntax.